parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java

```diff
@@ -81,6 +82,16 @@
 
   private static final Log LOG = Log.getLog(ParquetMetadataConverter.class);
 
+  private final boolean useSignedStringMinMax;
+
+  public ParquetMetadataConverter() {
+    this.useSignedStringMinMax = false;
+  }
+
+  public ParquetMetadataConverter(Configuration conf) {
+    this.useSignedStringMinMax = conf.getBoolean("parquet.strings.use-signed-order", false);
```

Comment:
I'd call this setting ```parquet.read.use-signed-order-for-unsigned-types.enabled```
see: https://github.com/apache/parquet-mr/blob/master/parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetInputFormat.java